packages/provider/src/transports/wallet-request-handler.ts

```diff
@@ -831,13 +826,17 @@ export class WalletRequestHandler implements ExternalProvider, JsonRpcHandler, P
 }
 
 export interface WalletUserPrompter {
+  getDefaultChainId(): number
```

Comment:
btw, just wondering, why do we need `getDefaultChainId()` on the prompter? The idea of the prompter interface is that these are methods which require "user input" / "user confirmation", so the UI will actually prompt the user to proceed or decline